dotcom-rendering/src/client/ophan/recordInitialPageEvents.ts

```diff
@@ -11,7 +11,7 @@ export const recordInitialPageEvents = async (): Promise<void> => {
 
 	// We wait for the load event so that we can be sure our assetPerformance is reported as expected.
 	window.addEventListener('load', function load() {
-		recordPerformance();
 		window.removeEventListener('load', load, false);
+		void recordPerformance();
```

Comment:
because the promise is not awaited, I do not think this changes the fact that the event listener is removed before any new even can be sent, but this seems like a more correct ordering.